src/main/java/domain/pages/HomePage.java

```diff
@@ -10,46 +12,53 @@
     Boolean result;
     String regex = "^[0-9]{5}(?:-[0-9]{4})?$";
 
-    @FindBy (xpath = "(//a[@href = '/representatives'])[1]")
+    @FindBy (xpath = "//a[@title='Home']")
```

Comment:
can you explain me whta this regex does?

Reply:
Abi siz benim son push versiyona mi bakiyorsunuz, cunku ben bu regex i kullanmadim son versionda. Bu regex diyor ki:
^-> string in baslangici,
[0-9]{5} -> 0 ile 9 arasi rakamlardan olusan bes adet rakam olacak,
(?:...)-> bu kismi alma diyoruz [0-9]{4}->0 ile 9 arasi rakamlardan olusan dort adet rakam,
?-> son dort rakam optional diyoruz,
$-> end of string